server/server/settings.py

```diff
@@ -26,7 +26,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config("DEBUG", default=False, cast=bool)
 
-ALLOWED_HOSTS = config("ALLOWED_HOSTS", cast=Csv())
+ALLOWED_HOSTS = ['*'] # config("ALLOWED_HOSTS", cast=Csv())
```

Comment:
The game plan for deployment is that this will run in Kubernetes with the communications mediated via Istio. This won't be using a virtual-hosts style setup where multiple domains are served from a single IP and distinguished by the host header... it's a dedicated cluster for a single system. 
We'll be hosting both parts of the system and routing traffic based on paths. 
To the degree that decisions need to be made based on the host header, we'd want that to be happening in [Istio's gateway configuration](https://istio.io/latest/docs/reference/config/networking/gateway/).